src/components/message/message-card/MessageCardTools.tsx

```diff
@@ -31,27 +31,30 @@ const COPY_NOTIFICATION_TEXT = 'Text copied to the clipboard!';
 
 const JSON_COPY_OPTIONS = ['body', 'fields'] as const;
 
-export type MessageCardToolsConfig = {
+export type MessageCardToolsProps = {
 	message: EventMessage;
-	parsedMessage: ParsedMessage | null;
-	messageViewType: MessageViewType;
-	toggleViewType: (viewType: MessageViewType) => void;
 	isBookmarked: boolean;
 	toggleMessagePin: () => void;
-	isScreenshotMsg: boolean;
 	isEmbedded?: boolean;
```

Comment:
After we make toggleMessagePin optional we can delete isEmbedded prop.
and we render bookmark button based on whether we have toggleMessagePin fn or not